### PR TITLE
fix: minish resync_delay for e2e test 

### PR DESCRIPTION
**Pull request by starsz**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

#922
___
### Bugfix
- Description

Because the resync_delay's default value is 5.
So apisix will sleep 5 or more seconds in sync data from etcd.
This will cause the e2e test to fail.

See more details: https://github.com/apache/apisix/pull/2977

- How to fix?

Set the resync_delay to 0.5 seconds.
___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: api/test/docker/apisix_config.yaml [hunk 1]
```diff
@@ -42,6 +42,7 @@ apisix:
 
 nginx_config:
   error_log_level: "debug"
+  worker_processes: "1"
```
Comment (membphis) [inline api/test/docker/apisix_config.yaml:45]: need a comment, only for debug now

File: api/test/e2e/global_rule_test.go [hunk 2]
```diff
@@ -79,7 +80,7 @@ func TestGlobalRule(t *testing.T) {
 			ExpectStatus:  http.StatusOK,
 			ExpectBody:    "hello world",
 			ExpectHeaders: map[string]string{"X-VERSION": "1.0"},
-			Sleep:         sleepTime,
+			Sleep:         longSleepTime,
```
Comment (membphis) [inline api/test/e2e/global_rule_test.go:83]: need some comment too.

Comment (membphis) [reply api/test/e2e/global_rule_test.go:83]: And please update other similar points

Comment (starsz) [reply api/test/e2e/global_rule_test.go:83]: OK. I had added the comment at base.go already.

https://github.com/starsz/apisix-dashboard/blob/e836046319fe37920f0d65c5a48d19aaa96b7d54/api/test/e2e/base.go#L185-L186

File: api/test/docker/apisix_config.yaml (hunk 1 above)
Comment (starsz) [reply api/test/docker/apisix_config.yaml:45]: Only for debug? Maybe I will add it to master if it can make the e2e test passed.

Comment (membphis) [reply api/test/docker/apisix_config.yaml:45]: we need to find the real bug. 

I think we should remove this field in E2E case.

**Comment by membphis** [comment]
@starsz CI failed again. please take a look

**Comment by starsz** [comment]
Hi, @membphis @nic-chen  Please have a look. I found the cause of the problem.

**Review by nic-chen — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**